radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/messaging/MessageCentralImpl.java

```diff
@@ -163,6 +171,11 @@
 	}
 
 	private Optional<MessageFromPeer<Message>> processInboundMessage(InboundMessage inboundMessage) {
+		final var messageQueuedTime = System.currentTimeMillis() - inboundMessage.receiveTime();
```

Comment:
Please, use `Time` class as the source of the time. This will make timestamping consistent across the codebase.

Reply:
done